OpenRA.Game/Support/ExceptionHandler.cs

```diff
@@ -56,9 +56,9 @@ static StringBuilder BuildExceptionReport(Exception ex)
 			return BuildExceptionReport(ex, new StringBuilder(), 0);
 		}
 
-		static StringBuilder AppendIndentedFormatLine(this StringBuilder sb, int indent, string format, params object[] args)
+		static StringBuilder AppendIndentedFormatLine(this StringBuilder sb, int indent, string message)
```

Comment:
After removing the format parameter the method nme AppendIndentedFORMATLine seems rather misleading.

Reply:
Valid point. Remove Format?

Reply:
Seems like the right thing to do